src/Microsoft.AspNetCore.SignalR.Client/HubConnectionExtensions.cs

```diff
@@ -89,6 +90,11 @@ public static ReadableChannel<TResult> Stream<TResult>(this HubConnection hubCon
             return outputChannel.In;
         }
 
+        private static void On(this HubConnection hubConnetion, string methodName, Type[] parameterTypes, Action<object[]> handler)
```

Comment:
Do we need to expose this overload? Shouldn't people use the generic overloads rather than pass in the parameter type array?